app/helpers/pages_helper.rb

```diff
@@ -33,6 +33,15 @@ def plugin_title(plugin)
     "#{plugin.name}#{detail}"
   end
 
+  def plugin_icon(plugin)
+    registered = {
+      action: 'hand-rock-o',
```

Comment:
they're [font-awesome](https://fortawesome.github.io/Font-Awesome/icons/) names for icons. this method provides the icon from the hash if its known or just gives it the stacked cubes icon if it's not in it.